apps/src/StudioApp.js

```diff
@@ -1,5 +1,14 @@
-// Globals:
-//   Blockly
+/* jshint
+  strict: true,
+  curly: true,
+  funcscope: true,
+  newcap: true,
+  nonew: true,
```

Comment:
This option prohibits the use of constructor functions for side-effects.  Actually caught something in this file, I'll comment on it specifically.